pyproject.toml

```diff
@@ -19,13 +20,21 @@ keywords = ['shipchain', 'transmission']
 [tool.poetry.dependencies]
 python = "^3.6"  # Compatible python versions must be declared here
 
+aws-requests-auth = "~0.4"
+boto3 = "~1.9"
+cryptography = "~2.5"
 django = "^2.2.4"
+django-enumfields = "==0.10.0"
 djangorestframework = "^3.9.1"
+djangorestframework-jsonapi = "~2.5"
+python-dateutil = "~2.7.5"
 requests = "^2.21"
 
 [tool.poetry.dev-dependencies]
 bandit = "^1.5"
 coverage = "~4.5.1"
+django-influxdb-metrics = {git = "git://github.com/lwbco/django-influxdb-tagged-metrics.git", rev = "add895558ed6ab59c621b19beae026f8933d4848", develop = true}
```

Comment:
I am working on publishing this package so it can be moved back to be a normal dependency.